.jshintrc

```diff
@@ -1,5 +1,5 @@
 {
-	"esversion": 6,
+	"esversion": 11,
```

Comment:
I did some research, and this actually means [the minimum ECMAScript-supported version](https://jshint.com/docs/options/#esversion). The [11th version was introduced in June 2020](https://262.ecma-international.org/11.0/index.html), and it supports optional property chaining (which we already use in many places of the code).